apps/user-profile/src/state/reducers/onboardingReducer.js

```diff
@@ -3,9 +3,11 @@ import { onboardingActions } from './../actions';
 import { featureFlags } from './../../utils/featureFlags';
 
 export const initialOnboardingState = {
-    showIntroductionModal: true,
-    showPlatformOnboardingModal: false,
-    user: {
+    
+    showIntroductionModal: false,
+    showPlatformOnboardingModal: true,
+    showCloseModal: false,
```

Comment:
Revert these to the default setting: 
`showIntroductionModal: true,`
`showPlatformOnboardingModal: false,`